src/AddImageChecks.cpp

```diff
@@ -351,7 +351,15 @@ Stmt add_image_checks_inner(Stmt s,
         }
 
         // Check that the region passed in (after applying constraints) is within the region used
-        debug(3) << "In image " << name << " region touched is:\n";
```

Comment:
This old code is confusing... did there use to be a debug(3) inside the loop over j below too?

Reply:
It appeared to be a weird dangling thing, presumably left over from some earlier refactoring